### DISCUSSION TITLE
Allow transactions from "expense account" to "asset account"

### DISCUSSION THREAD

**Discussion by gthbusrr** [firefly-iii/firefly-iii#4312] [Firefly III]
**Description**
- I would like to track refunds, paybacks from friends, etc and not having it counted as "expenses" (for the fund that I loaned out or paid) nor "income" (for the refund or the payback).

**Solution**
- Allowing transactions from "expense account" to "asset account"
OR
- Allow convert "expense account" as "liabilities" or creating "libilities" in transactions like how "expense account" is created

**What are alternatives?**
- Create transactions with "friend's name" or "business name" as "expense account", tag "Debt - {friend's name}" for loans
- Create income with "friend's name" or "business name" as "revenue account", tag "Debt - {friend's name}" for loans
- See how much friend owe you in tag
- No way to merge transactions

**Additional context**

**Bonus points**
_**I MUST SAY, THANK YOU SO MUCH FOR DEVELOPING FIREFLY III, I HAVE BEEN TRACKING MY SPENDINGS VIA SPREADSHEET SINCE GRADE 12... DISCOVERING FIREFLY III REALLY CHANGED MY LIFE. THANK YOU SO MUCH!**_

**Comment by JC5** [comment]
Thanks for the compliments, it's much appreciated.

Keeping track of outgoing loans is work ongoing and I hope to introduce something in a new version. I'll put something in the documentation to show you what I plan to do. It'll be some time to work this out properly, but at least you'll know what I'm up to!

- https://docs.firefly-iii.org/firefly-iii/faq/financials/#what-do-i-do-with-people-who-pay-me-back

**Comment by gthbusrr** [comment]
I've read the document (Great Work), but changing the amount spent will affect the sum in accounts (e.g. Paid for friend by Credit Card, I paid $100 by credit card, so the balance should be -$100 in credit card, $50 to the merchant and $50 to my bank (payback from friends)). So allowing transfer of fund from expense account to asset account would be great!

Anyway, glad to know that it's being taking care of, thank you very much for your effort!

**Comment by gthbusrr** [comment]
After discussing with an accountant, she gave the following suggestions.

- Allowing split to two account types: Asset Account to both "Expense Account (EA)" (Like the restaurant) and "Asset Account (AA)" (Friend). Thus, when they pay you back, you can transfer from his account (AA) to your bank (AA)
- Allow creation of AA in transactions.

And also, my suggestion:
- Add a "Net Balance" for tag view, so that one can track the net transaction amount of all tags, including income and expense.

Thank you very much!

**Comment by JC5** [comment]
Each tag already has that. Just visit /tags.

**Comment by gthbusrr** [comment]
> Each tag already has that. Just visit /tags.

Currently, I see **Sum of withdrawals** -HK$376.00 and **Sum of deposits** HK$376.00, but I would love to see **Net Balance**: HK$0.00

Thank you very much 👍  :)

![Screenshot_2021-01-18 All transactions for tag Debt - Lai Chi Shing between January 1, 2021 and January 31, 2021 » Tags » F](https://user-images.githubusercontent.com/77537216/104878197-68c79500-5953-11eb-970e-3c9f3eb1a234.png)
